woocommerce/woocommerce-android
# Issue/12539 blaze push notification navigation

Closes: #12539


### Description

When a Blaze campaign status update push notification is received upon tapping on it: 
- Load the campaign list screen and open the campaign detail screen when tapping on a single notification
- If notifications are grouped, only open campaign list

### Testing information
1. Make a clean install of the app (we need this to ensure the FCM token is fresh for `20.2` version of the app and Blaze push notifications are enabled)
2. Create a Blaze campaign. 
3. Go to the Slack channel #wp-promote-moderation and select you campaign. Approve it. This will trigger an approved campaign push.
4. Tap on the notification and notice how the campaign detail screen is loaded correctly. 
5. Navigate back from campaign detail and check that you first land on campaign list screen and finally in the more menu tab. 

Repeat the steps above to generate at least 2 Blaze push notifications (tip:cancelling/Stopping the campaign from the detail screen webview will also trigger a push notification)

1. Once you have at least 2 notifications, click on the "grouped" notification and notice how the app open in the campaign list screen. 

Finally, the smoke test will be done to ensure that new order notifications and product review notifications work as expected. 

### The tests that have been performed

- Blaze push notifications tested (single and grouped)
- New Order notification tested.
- New review notification tested. 

### Images/gif


https://github.com/user-attachments/assets/d6df0026-1371-4b91-8adb-0680293d38b5


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
